lib/improver/tests/nbhood/test_CircularNeighbourhood.py

```diff
@@ -37,8 +37,8 @@
 from iris.tests import IrisTest
```

Comment:
All of the /nbhood tests need to move to /nbhood/nbhood now you have created an nbhood directory in /lib/improver